editor/src/main/java/org/pentaho/pms/ui/QueryBuilderDialog.java

```diff
@@ -420,7 +421,10 @@ private void executeSQL(DatabaseMeta databaseMeta, String sql) {
       if (columnsMap != null){
         for (int i = 0; i < rm.size(); i++){
           ValueMetaInterface value = rm.getValueMeta(i);
-          value.setName((String)columnsMap.get(rm.getValueMeta(i).getName()));
+          String colName = (String)columnsMap.get(rm.getValueMeta(i).getName());
+          if(!StringUtil.isEmpty(colName)) {
```

Comment:
This may be a question for @smmribeiro or @renato-s, is there a reason to keep using and maintaining an in house `StringUtils` if, at least for this case, it can be replaced with `org.apache.commons.lang.StringUtils`, that is already used in many places in the code base?

Reply:
Our class probably predates the Apache one...
A quick search, renders some 8 StringUtil/StringUtils classes on our whole solution and the available methods are not always the same and even when the same method exists, the implementation can vary... basically they grew put of necessity...
In this specific case, I wouldn't see a problem using the "lang3" version...